adam-core/src/test/scala/org/bdgenomics/adam/util/ADAMFunSuite.scala

```diff
@@ -25,6 +25,7 @@ trait ADAMFunSuite extends SparkFunSuite {
   override val properties: Map[String, String] = Map(("spark.serializer", "org.apache.spark.serializer.KryoSerializer"),
     ("spark.kryo.registrator", "org.bdgenomics.adam.serialization.ADAMKryoRegistrator"),
     ("spark.kryoserializer.buffer.mb", "4"),
-    ("spark.kryo.referenceTracking", "true"))
+    ("spark.kryo.referenceTracking", "true"),
+    ("spark.hadoop.parquet.task.side.metadata", "false"))
```

Comment:
Task side is on the cluster nodes, client side is where the Spark client is running. The reason that task side is the default now for reading metadata is to avoid situations where a large number (thousands) of Parquet files are being processed and the client has to read all the footers before the job can start. See https://issues.apache.org/jira/browse/PARQUET-139

Row group filtering will occur in both cases (client side and task side) - I was wrong to say above that it only occurs on the client side. So I've made a minor adjustment to the code to remove this change, and keep the Parquet default of task side row group elimination. I used the debugger to confirm that row group elimination was still happening in the test.